.github/workflows/official-pr.yml

```diff
@@ -0,0 +1,63 @@
+name: Create official images PR
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+    paths:
+      - "**/Dockerfile"
+      - "**/architectures"
+      - "**/docker-entrypoint.sh"
+      - "generate-stackbrew-library.sh"
+      - "functions.sh"
+
+jobs:
+  pr:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'nodejs' && github.event.pull_request.merged_by != ''
+
+    steps:
+      - name: Checkout the docker-node repo
+        uses: actions/checkout@v2
+        with:
+          path: docker-node
+          fetch-depth: 50
+
+      - name: Checkout the official-images repo
+        uses: actions/checkout@v2
+        with:
+          path: official-images
+          repository: docker-library/official-images
+
+      - name: Generate Stackbrew for diff
+        run: |
+          cd docker-node
+          ./generate-stackbrew-library.sh > ../official-images/library/node
+
+      - name: Create PR in official-images
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_API_TOKEN }}
+          push-to-fork: nodejs-github-bot/official-images
+          path: official-images
+          branch: autopr
+          branch-suffix: short-commit-hash
```

Comment:
```suggestion
          branch-suffix: ${{ github.event.pull_request.number }}
```

Reply:
The suffix field is more of an enum and behaves this way: https://github.com/peter-evans/create-pull-request/blob/da928d5fcce0b0dd0c34642f77e862dc9cf76d8b/dist/index.js#L326

However, we can just remove the `branch-suffix` option and add the pr number to `branch`

Reply:
Ah, OK. Not a big one, it's fine as-is